math/wxMaxima/Portfile

```diff
@@ -36,7 +35,8 @@ depends_lib-append  port:${wxWidgets.port} \
 
 # Note: the advantage of path:bin/cmake:cmake over port:cmake is that it also allows e.g. cmake-devel
 # Note: the cmake port group seems to have issues with application packages
-depends_build-append path:bin/cmake:cmake
+depends_build-append path:bin/cmake:cmake \
+                     path:bin/cppcheck:cppcheck
```

Comment:
Why is a path dependency used here? Is there an alternative port to `cppcheck` in Macports?

Reply:
Well there could be a dev version one day and people might have it installed. If I remember right on my previous PR you said that path is preferable. Is the rule to use path only if there exists a variant of the package in question today? I must say I wouldn't find that a very plausible strategy. Or is cmake special because it is so widely used?

Reply:
The path style is not preferable. If there is a `-devel` version or another port can satisfy the dependency, then it should be used. Since no other port can satisfy the dependency in this case, the port style should be used.

Reply:
Thanks for your detailed reviews - I still have to learn a lot about Macports. I will do your changes and push later today.

Reply:
I am also not sure that `cppcheck` should be added here at all. Do you think that a typical user (i.e., non-wxMaxima developer) would be interested in checking the source code during compilation? If so, sure you can add it - but why not do that when you update to a version where it's actually used. Alternatively, it appears you can just use the `WXM_USE_CPPCHECK` flag to switch it off.

Reply:
> I am also not sure that `cppcheck` should be added here at all. Do you think that a typical user (i.e., non-wxMaxima developer) would be interested in checking the source code during compilation? If so, sure you can add it - but why not do that when you update to a version where it's actually used. Alternatively, it appears you can just use the `WXM_USE_CPPCHECK` flag to switch it off.

As far as I can tell in current Maxima master the tests are run by default during a build. In general I think it is a good idea to run such tests (they have been added for a reason). But if the Macports team thinks it is preferable to nun run such tests - say to trust the Maxima CI, I can indeed switch them off.
Note: the 20.04 version used currently does not use this, I added it because I have it installed so it is easy to miss this dependency. I plan to do further updates soon.